### ISSUE TITLE
Apply phetioFeatured

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab#174]
from #76, we are at the point where we want to implement a first pass at this now. @kathy-phet over to you. I'm looking forward to our meeting tomorrow!

**Comment by zepumph** [comment]
This seemed to go pretty smoothly, @kathy-phet took less than an hour to go through this sim. She then passed the overrides from the console over to me, and I committed them.

One thing I ran into is that I needed to regenerate the baseline after pasting the overrides. This was because may linked elements updated their phetioFeatured values from this. I faintly remember working on this with @samreid and @chrisklus, but it was uncomfortable to have api mismatches after adding in the overrides. We should probably make sure to document that. It also increases the complexity of having designers test their changes to phetioFeatured.
